tests/models/test_agent.py

```diff
@@ -38,3 +44,43 @@ def test_get_agent_by_name_not_found(db_session):
     db_session.commit()
 
     assert get_agent_by_name(db_session, 'different') is None
+
+
+def test_list_all_agents_size(db_session):
+    db_session.add(Agent(name='test'))
+    db_session.commit()
+
+    assert len(list_all_agents(db_session)) == 1
+
+    db_session.add(Agent(name='test2'))
+    db_session.commit()
+
+    assert len(list_all_agents(db_session)) == 2
+
+
+@pytest.mark.asyncio
+async def test_agent_connection_status(db_session, test_client):
+    agent = Agent(name='test')
+    db_session.add(agent)
+    db_session.flush()
+    agent_id = agent.id
+    db_session.commit()
+
+    websocket_manager = WebSocketManager()
```

Comment:
Using `WebSocketManager` like this in tests makes for flaky tests. It's a singleton which connections won't get removed after test completion. I've made a fixture for it in `AP-12`

Reply:
So leave it for now?

Reply:
Might as well delete the test then. We can't rely on its outcome as is, so I don't think it makes sense to add a new flaky test to develop (being aware that several existing tests have the same problem).